aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -98,6 +98,8 @@
       path: /tmp/slack.sh
     - content: |
           0 * * * * curl -X POST -H "Content-type: application/json" --data '"'"'{"$(ec2metadata --instance-type) instance $(ec2metadata --instance-id) running... \\n Batch [$UID] completed and instance of type $(ec2metadata --instance-type) is still running in $REGION since last $(($(($(date +%s) - $(cat /tmp/.starttime))) / 3600)) Hour $(($(($(date +%s) - $(cat /tmp/.starttime))) / 60)) Minute."}'"'"
+          */10 * * * * /home/ubuntu/beam_stuck_guard.sh
+          
       path: /tmp/slack_notification
```

Comment:
I do not understand what this part do. 
Our slack notifications do not contain text from first row of file - I am not sure it is working at all, or, if it is - where to look for these notifications.
The added row should sent slack notifications too?
It seems the content of 'beam stuck guard' should kill beam if it is stuck, if so, this is not a place for such script.

Reply:
I suggest to remove old and not working part from slack-notification script and rename the script into something appropriate to its functionality - 'cron-tasks' or something like that